pydicom/util/codify.py

```diff
@@ -185,6 +197,12 @@ def code_sequence(
     # Code line to create a new Sequence object
     if name_filter:
```

Comment:
This check is not needed, as there is always a filter. If there wasn't, this case should be handled, otherwise `seq_var` would be undefined.

Reply:
I don't mind leaving that in - similar optional parameters have been used directly in tests.  It wasn't here, but perhaps may be in future.

Reply:
Hm, but as I wrote - if it were optional, the code would crash if it were not set...